### PR TITLE
percy config

### PR DESCRIPTION
**Pull request by dmytrofirsov**

### CODE REVIEW

**Review by Vita-Karas — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -472,6 +490,9 @@ workflows:
                         branches:
                             only:
                                 - main
+            - bits-percy-visual-test:
+                  requires:
+                      - bits-build
```
Comment (Blakesalot) [inline .circleci/config.yml:495]: Should we add the same branch filter as the one used by `bits-visual-test` starting at line 489?

File: packages/bits/.percy.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  # (per-snapshot Percy CSS is concatenated to the end of this value)
+  # Details: https://docs.percy.io/docs/percy-specific-css
+  percyCSS: |
+    input, textarea { 
+      caret-color: transparent;
+    }
+    * {
+      cursor: none !important;
+    }
+  # WE DON'T CARE ABOUT THE CONFIGS BELOW
```
Comment (Blakesalot) [inline packages/bits/.percy.yml:18]: Just curious, why are they here if we don't care about them? Maybe this information could be included in the comment?

**Review by Blakesalot — CHANGES_REQUESTED** [review]

File: packages/bits/.percy.yml (hunk 2 above)
Comment (dmytrofirsov) [reply packages/bits/.percy.yml:18]: Theoretically, those options could be used for another way of taking snapshots - taking local screenshots and bulk-sending them to percy. I decided to leave them there for future reference, I think I'll be reviewing it in the next sprint in the scope of other percy tasks, so no worries.

File: .circleci/config.yml (hunk 1 above)
Comment (dmytrofirsov) [reply .circleci/config.yml:495]: makes sense, let me change that

**Review by Blakesalot — APPROVED** [review]

**Pull request merged**